VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/custom/CustomSelectViewAction.java

```diff
@@ -0,0 +1,38 @@
+package org.edx.mobile.test.feature.custom;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.view.View;
+
+import org.edx.mobile.view.custom.CustomSelectView;
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isAssignableFrom;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static org.hamcrest.Matchers.allOf;
+
+/**
+ * Defines the custom {@link ViewAction} implementation to select an item
+ * from our custom implementation of spinner i.e. {@link CustomSelectView}.
+ */
+public class CustomSelectViewAction {
+    public static ViewAction selectSpinnerItem(final int index) {
+        return new ViewAction() {
```

Comment:
This should probably be wrapped with `ViewActions.actionWithAssertions()`, which should run all the globally defined assertions on the view when performing the action.